docs/styleguides/python.md

```diff
@@ -197,11 +197,11 @@ from test_util.helpers import (CI_CREDENTIALS,
 
 * TODO Comment Format
     * Follow the rules on [when to write TODO comments](https://google.github.io/styleguide/pyguide.html?showone=TODO_Comments#TODO_Comments)
-    * Format required will be `TODO (active-developer) [JIRA] - Brief one line description`
+    * Format required will be `#TODO (active-developer) JIRA_ID: Brief one line description`
 
     ```
-    # TODO: (skumaran) DCOS_OSS-952 - Create a flake8 plugin to enforce TODO format.
-    ````
+    #TODO (skumaran) DCOS_OSS-952: Create a flake8 plugin to enforce TODO format.
```

Comment:
> My proposal and suggestion is that we want to use the JIRA ticket with the active developer name.

Then why not just a JIRA? Jira owner can change, developer can move to a different project/team.